neuro_san/service/watcher/main_loop/storage_watcher.py

```diff
@@ -49,14 +48,16 @@ def __init__(self,
         self.manifest_path: str = manifest_path
         self.update_period_seconds: int = update_period_seconds
         self.logger = logging.getLogger(self.__class__.__name__)
-        self.updater = threading.Thread(target=self._run, daemon=True)
-        if self.use_polling:
-            poll_interval: int = self.compute_polling_interval(update_period_seconds)
-            self.observer = RegistryPollingObserver(self.manifest_path, poll_interval)
-        else:
-            self.observer = RegistryEventObserver(self.manifest_path)
+        self.updater_thread = threading.Thread(target=self._run, daemon=True)
+
+        poll_interval: int = self.compute_polling_interval(update_period_seconds)
+        self.storage_updaters: List[StorageUpdater] = [
+            RegistryStorageUpdater(network_storage_dict, manifest_path, poll_interval)
+            # We will eventually have more here
+        ]
+
         self.server_status: ServerStatus = server_status
-        self.go_run: bool = True
+        self.keep_running: bool = True
```

Comment:
Rename that reads better in the while loop below.
It did not have anything to do with launching go-runner jobs :)